submissions/Zakkarat/a-tiny-JS-world/index.js

```diff
@@ -0,0 +1,78 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const dog = {
+  name: "Paolo",
+  species: "dog",
+  gender: 'male',
+  legs: 4,
+  hands: 0,
+  saying: "Bark!"
+}
+const cat = {
+  name: "Sophie",
+  species: "cat",
+  gender: 'female',
+  legs: 4,
+  hands: 0,
+  saying: "Mur-meow?"
+}
+const human = {
+  name: "Pablito",
+  species: "human",
+  gender: 'male',
+  legs: 2,
+  hands: 2,
+  saying: "It seems that we got a strange rat here..."
+}
+const rat = {
+  name: "Twitch",
+  species: "rat",
+  gender: 'male',
+  legs: 2,
+  hands: 2,
+  saying: "Oh, HELLO!"
+}
+const cat_woman = {
```

Comment:
Use `camelCase`  for that. IMO `catWoman` will be better.

Reply:
Done!